subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheProjectReportIntegTest.groovy

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.configurationcache
+
+class ConfigurationCacheProjectReportIntegTest extends AbstractConfigurationCacheIntegrationTest {
+
+    def setup() {
+        buildFile << """
+            apply plugin: 'project-report'
+        """
+    }
+
+    def "configuration cache for Project Report plugin task '#task' on empty project"() {
+        given:
+        settingsFile.createFile()
```

Comment:
💄 I'd move this line to `setup` above and write it like `settingsFile << ""` for similarity to how `buildFile` is set.

Reply:
Turns out I had no reason to create a settings file, just removed it